From 7f12334981dc9b268de92a306f6b5443eff1ad3e Mon Sep 17 00:00:00 2001
From: Meng Li <Meng.Li@windriver.com>
Date: Mon, 27 Feb 2023 17:10:06 +0800
Subject: [PATCH] m7-sample: remove the code of enabling CortexA53
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In parallel secure boot mode, it doesn’t need to bring up A53
core with M7 core any longer.

Upstream-Status: pending

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 src/main.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index d130977..66f6e70 100644
--- a/src/main.c
+++ b/src/main.c
@@ -131,6 +131,5 @@ static void enable_a53(void)
 
 int main(void)
 {
-	enable_a53();
 	return 1;
 }
-- 
2.36.1

